pkg/apis/acme/v1/types_issuer.go

```diff
@@ -317,6 +317,9 @@ type ACMEChallengeSolverHTTP01IngressTemplate struct {
 	// will override the in-built values.
 	// +optional
 	ACMEChallengeSolverHTTP01IngressObjectMeta `json:"metadata"`
+
+	// OverrideNginxIngressWhitelistAnnotation add description here
```

Comment:
Can we add some description for this field? 👍

Reply:
Seems like that one slipped through the cracks, I'll update the PR with some actual descriptions :)